src/components/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react'
+
+export class Footer extends Component {
```

Comment:
If you are exporting at the bottom of your component, you will want to remove it from up here.  Having two exports on the same component may throw errors.  Please remove these extra exports on all of your components and pages.
```suggestion
class Footer extends Component {
```